setup.py

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env python
```

Comment:
Yeah, but the file wasn't executable.  And I thought we never install it by running either `./setup.py` or `python setup.py` manually.  If there's case for that - I'll return it back